### PR TITLE
doc(0.4.x): Folders

### PR DESCRIPTION
**Pull request by miller-productions**
Add new section, Folders, to 0.4.x doc.

### CODE REVIEW

**Comment by miller-productions** [comment]
This is essentially a small starting point for addition to 0.4.x doc. Deliberately kept the update constrained to see if it is the type of thing everyone thinks would add value. Would possibly expand on it further in future pull requests by going down a bit deeper into descriptions of sub-folders, i.e. like what was done in the [0.3.x doc](http://meanjs.org/docs/0.3.x/#folder-structure).

File: _includes/docs/0.4.x/folders.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  </p>
+  <h2>public</h2>
+  <p>
+    This folder contains all the static front-end files used by the app to be served out. It includes elements built from your application's source as well as third-party external libraries required by your application.
+  </p>
+  <h2>scripts</h2>
+  <p>
+    This folder is home to various helper scripts used during development, administration and operation of your application.
+  </p>
+ </section>
```
Comment (codydaig) [inline _includes/docs/0.4.x/folders.html]: New Line

**Comment by codydaig** [comment]
Looks good to me. If you can put a new line at the end of the new file, I'll merge this in.

**Comment by miller-productions** [comment]
Thanks @codydaig, will do.

**Comment by miller-productions** [comment]
Facepalm :neutral_face::hand::palm_tree:

Fixed.

:shipit:

**Pull request merged**